picasso/src/test/java/com/squareup/picasso/DeferredRequestCreatorTest.java

```diff
@@ -54,48 +50,13 @@
     initMocks(this);
   }
 
-  @Config(reportSdk = 12)
-  @Test public void initAddsAttachListenerWhichDefersLayoutListener() {
```

Comment:
Woof, this is embarrassing. I did delete those while trying to isolate an issue and forgot to put them back -_-

That said, min SDK of the project went to 14 in the gradle change. Should that still be something older? (I don't know what it was before)